modules/thermal_hydraulics/doc/content/source/ics/SpecificTotalEnthalpyIC.md

```diff
@@ -1,20 +1,16 @@
 # SpecificTotalEnthalpyIC
 
-!alert construction title=Undocumented Class
-The SpecificTotalEnthalpyIC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /ICs/SpecificTotalEnthalpyIC
 
-## Overview
-
-!! Replace these lines with information regarding the SpecificTotalEnthalpyIC object.
-
-## Example Input File Syntax
+The cross-sectional area variable, [!param](/ICs/SpecificTotalEnthalpyIC/A),
+is usually set by the [Component](syntax/Components/index.md).
+For single-phase single-fluid thermal-hydraulics, the volume fraction,
+[!param](/ICs/SpecificTotalEnthalpyIC/alpha), should remain
+equal to one.
 
-!! Describe and include an example of how to use the SpecificTotalEnthalpyIC object.
+!alert note
+This initial condition is usually added to the `Simulation` by the `flow model`, based on the parameters
```

Comment:
I'm not used to seeing code formatting with a space, `flow model`

Reply:
should it just be "flow model", no code formatting, or `flow_model`?

Reply:
it is code but it is also code that is under the hood and undocumented
so it could be `FlowModel` but then if you search for that in the documentation you cant find it yet..

I think I ll just try to have that page ready in the near future